### PR TITLE
Feature/implementation gherkin files alan

### PR DESCRIPTION
**Pull request by Amet015**

### CODE REVIEW

File: trello/api/features/acceptance/OrganizationAcceptance.feature [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+      | key      | value         |
+      | email    | (email)       |
+      | fullName | alan escalera |
+    And Sends request
+    And Should return status code 200
+    And Saves response as "organization_membersObject"
+    And Validates response body with
+      | key                    | value |
+      | memberships.memberType | None  |
+      | members.fullName       | None  |
```
Comment (Enrique-C) [inline trello/api/features/acceptance/OrganizationAcceptance.feature]: Review this body please, you would review the member that was be added

File: trello/api/features/acceptance/OrganizationAcceptance.feature [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    And Validates schema with "put_organization_members.json"
+    And Sets a "GET" request to "/organizations/organization_membersObject.id/members"
+    And Sends request
+    And Should return status code 200
```
Comment (Enrique-C) [inline trello/api/features/acceptance/OrganizationAcceptance.feature:25]: Where is then here?

File: trello/api/features/acceptance/OrganizationAcceptance.feature [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+    #Post Condition
+    And Sets a "DELETE" request to "/organizations/organizationObject.id"
+    And Sends request
+    And Should return status code 200
```
Comment (Enrique-C) [inline trello/api/features/acceptance/OrganizationAcceptance.feature]: Add a code line at the last

File: trello/api/features/functional/CardFuntional.feature [hunk 4]
```diff
@@ -246,3 +246,43 @@ Feature: Card
     And Sends request
     And Should return status code 200
 
+
+  Scenario: Creates a new checklist on a card
```
Comment (Enrique-C) [inline trello/api/features/functional/CardFuntional.feature]: It is a Duplicated

File: trello/api/features/functional/OrganizationFunctional.feature [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+      | key         | value            |
+      | displayName | new Organization |
+    And Sends request
+    And Should return status code 200
+    And Saves response as "organizationObject"
+    When Sets a "POST" request to "/organizations/organizationObject.id"
+      | key         | value                    |
+      | displayName | new Name of Organization |
+    And Sends request
+    And Should return status code 200
```
Comment (Enrique-C) [inline trello/api/features/functional/OrganizationFunctional.feature]: Where is Then?

File: trello/api/features/functional/OrganizationFunctional.feature [hunk 6]
```diff
@@ -0,0 +1,18 @@
...
+    And Saves response as "organizationNameObject"
```
Comment (Enrique-C) [inline trello/api/features/functional/OrganizationFunctional.feature]: Add a code line at the last please

File: trello/api/features/smoke/CardSmoke.feature [hunk 7]
```diff
@@ -37,3 +37,38 @@ Feature: Card
...
+    When Sets a "DELETE" request to "/cards/CardObject.id"
+    And Sends request
+    And Should return status code 200
+    And Validates response body with
+      | key    | value |
+      | Limits | None  |
+    And Validates schema with "delete_card_label_schema.json"
+    And Sets a "GET" request to "/cards/CardObject.id"
+    And Sends request
+    And Should return status code 404
```
Comment (Enrique-C) [inline trello/api/features/smoke/CardSmoke.feature]: Where is Then?

File: trello/api/features/smoke/CardSmoke.feature [hunk 8]
```diff
@@ -37,3 +37,38 @@ Feature: Card
...
+    And Sets a "DELETE" request to "/boards/BoardObject.id"
+    And Sends request
+    And Should return status code 200
```
Comment (Enrique-C) [inline trello/api/features/smoke/CardSmoke.feature]: Add a code line at the last please

File: trello/api/features/smoke/OrganizationSmoke.feature [hunk 9]
```diff
@@ -0,0 +1,28 @@
...
+      | displayName | new Organization |
+    And Sends request
+    Then Should return status code 200
+    And Saves response as "organizationObject"
+    And Validates response body with
+      | key         | value            |
+      | displayName | new Organization |
+      | desc        |                  |
+      | website     | None             |
+    And Validates schema with "Create_organization.json"
```
Comment (Enrique-C) [inline trello/api/features/smoke/OrganizationSmoke.feature]: Is it different to organization_schema.json?

File: trello/api/features/smoke/OrganizationSmoke.feature [hunk 10]
```diff
@@ -0,0 +1,28 @@
...
+    And Sends request
+    And Should return status code 200
+    #Post Condition
+    And Sets a "DELETE" request to "/organizations/organizationObject.id"
+    And Sends request
+    And Should return status code 200
+
+
+
+
```
Comment (Enrique-C) [inline trello/api/features/smoke/OrganizationSmoke.feature]: Review this please

File: trello/jsonfiles/put_organization_members.json [hunk 11]
```diff
@@ -0,0 +1,180 @@
...
+        }
+      ]
+    }
+  },
+  "required": [
+    "id",
+    "memberships",
+    "members"
+  ]
+}
```
Comment (Enrique-C) [inline trello/jsonfiles/put_organization_members.json]: Add a code line at the last

File: trello/api/features/functional/ListFunctional.feature [hunk 12]
```diff
@@ -2,7 +2,7 @@
 Feature: List
   Manages list of the board
 
-  Scenario: Move a list from a board to another one
+   Scenario: Move a list from a board to another one
```
Comment (Enrique-C) [inline trello/api/features/functional/ListFunctional.feature]: Review the indentation please

File: input_variables.json [hunk 13]
```diff
@@ -1,4 +1,4 @@
 {
-  "email": "Pedro.Quino@fundacion-jala.org",
+  "email": "Alesandro.Washingotn@fundacion-jala.org",
```
Comment (Enrique-C) [inline input_variables.json]: Create a new email please

File: trello/api/features/functional/CardFuntional.feature [hunk 14]
```diff
@@ -246,3 +246,44 @@ Feature: Card
...
+      | pos  | top                        |
+    And Sends request
+    Then Should return status code 200
+    And Saves response as "Card_AObject"
+    And Validates response body with
+      | key        | value                      |
+      | name       | Check List Card Functional |
+      | checkItems | []                         |
+    And Validates schema with "put_card_add_checklist.json"
+    And Sets a "GET" request to "https://api.trello.com/1/cards/Card_AObject.id/checklists"
```
Comment (Enrique-C) [inline trello/api/features/functional/CardFuntional.feature]: Do not use the URL here please, only the endpoint

File: trello/api/features/functional/OrganizationFunctional.feature [hunk 15]
```diff
@@ -1,7 +1,33 @@
-# Created by Juan Martinez at 09/02/2020
+# Created by Alan Escalera at 02/08/2020
 Feature: Organization
   As a regular user, it wants to manage a organization, and creates a organization.
 
+
```
Comment (Enrique-C) [inline trello/api/features/functional/OrganizationFunctional.feature]: Delete this code line

File: trello/jsonfiles/Create_organization.json [hunk 16]
```diff
@@ -0,0 +1,70 @@
+{
```
Comment (Enrique-C) [inline trello/jsonfiles/Create_organization.json]: Delete this file please

**Review by Enrique-C — APPROVED** [review]

File: trello/api/features/acceptance/OrganizationAcceptance.feature [hunk 17]
```diff
@@ -0,0 +1,30 @@
+# Created by Alan Escalera at 02/08/2020
```
Comment (LimbertVargas) [inline trello/api/features/acceptance/OrganizationAcceptance.feature]: please, remove this line

File: trello/api/features/functional/OrganizationFunctional.feature [hunk 18]
```diff
@@ -1,7 +1,32 @@
-# Created by Juan Martinez at 09/02/2020
+# Created by Alan Escalera at 02/08/2020
```
Comment (LimbertVargas) [inline trello/api/features/functional/OrganizationFunctional.feature]: please, remove this line

**Review by LimbertVargas — APPROVED** [review]

**Pull request merged**